model.py

```diff
@@ -348,7 +349,8 @@ class User(orm.Document):
               ('username', orm.StringField(required=True)),
               ('salt', orm.StringField(required=True)),
               ('hashed_password', orm.StringField(required=True)),
-              ('admin_regions', orm.ListField(orm.StringField()))]
+              ('admin_regions', orm.ListField(orm.StringField())),
+              ('admin_level', orm.StringField(required=True, default='REGION'))]
```

Comment:
So I've gone with the String Only method, but I'm not really sure how to validate on the ORM. can you point me in the right direction?